docs/UsersGuide.asciidoc

```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
 ISO_1_DECOMPRESS_URL=http://host/foo2.iso.xz and ISO_1=foo.iso
 
 
+== Asset cleanup ==
+
+Since assets consume a huge amount of disk space, openQA removes assets
+automatically according to configurable limits.
+
+=== Cleanup stragegy
+
+openQA frequently checks whether assets need to be removed according to
+the configured limit.
+
+To determine which assets can be removed, it looks by which job groups
+a certain asset is used. If at least one job within a certain job group
+is using an asset, the asset is considered to be used by that group.
+
+So an asset can be accounted to multiple groups. The assets table which is
+accessible via the admin menu shows these groups for each asset and also
+the latest job.
+
+If the size limit for assets by a certain group is exceeded, openQA will
+delete those assets:
+
+* Assets belonging to old jobs a preferred.
+* Assets belonging to jobs which are still scheduled or running will not be
+  considered at all.
+* Assets which also belong to another group which still has space left are
+  not considered as well.
+
+Assets which don't belong to any group are deleted after a configurable
```

Comment:
I think it is more common to use the full form of words for documentation, s/don't/do not/

Reply:
I know, I even used a 'will not' somewhere (even though won't is so much cooler). That one I've just overlooked.